icekit/project/settings/_base.py

```diff
@@ -414,18 +414,18 @@
 # given model/field, without generating additional unecessary thumbnails. This
 # is essential when using a remote storage backend.
 THUMBNAIL_ALIASES = {
-#     'app[.model][.field]': {
-#         'name-WxH': {
-#             'size': (W, H),
-#             ...,
-#         },
-#     },
+    # 'app[.model][.field]': {
+    #   'name-WxH': { 'size': (W, H), },
+    # },
     '': {
+        'author_portrait': {
+            'size': (360, 640),
+        },
```

Comment:
But we really have no idea what aliases templates from other apps might provide. If a project provides other templates (not authors app templates) or uses another app that has templates that use a global namespace thumbnail alias that matches with the authors app alias, they can define in their settings a global namespace alias with the same specs as the authors app namespace alias.